tasks/77_expressions-matter/expressions-matter.js

```diff
@@ -0,0 +1,17 @@
+// It works but I'm not sure if my approach to this problem is correct
+export function expressionMatter( a, b, c ) {
+  // eslint-disable-next-line prefer-rest-params
+  const sortedArgs = Object.values(arguments).sort((a, b) => a-b );
+  [a, b, c] = sortedArgs;
+
+  let result = [];
```

Comment:
you don't need result array as you can just write results to function without array.

also there is also some bad tests( with values `5, 6, 1` `1, 6, 1` `2, 6, 1` `6, 7, 1` `1, 8, 3` `1, 10, 1`) and you don't need to sort numbers, but it is good that you found a way to solve this so it can stay as it is

Reply:
Done